### PR TITLE
fix: server side delta generation docs update.

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: Title
Ticket: MEN-6288
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@kjaskiewiczz check if I captured all the requirements for delta to start.

File: 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md [hunk 1]
```diff
@@ -8,11 +8,29 @@ taxonomy:
...
-!!! Hosted Mender supports this feature in the Enterprise plan; however, its availability requires a tenant-specific activation.
-!!! If you are a customer interested in it, please get in touch with the [Hosted Mender support team](mailto:support@mender.io).
-
 ### Prerequisites
 
+#### Hosted Mender
+
+The automatic generation of delta artifacts on the server side with
+[hosted Mender](https://hosted.mender.io/ui/signup) works out of the box with the Enterprise plan, provided 
+you sign up for it. In order to do so, please please get in touch with the [hosted Mender support team](mailto:support@mender.io).
```
Comment (kjaskiewiczz) [inline 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md:17]: please please

Comment (merlin-northern) [reply 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md]: _Please Please Me_ -- The Beatles.

Comment (merlin-northern) [reply 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md]: fixed.

**Pull request merged**